oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/KafkaFetcherHandlerRegister.java

```diff
@@ -110,7 +118,22 @@ public KafkaFetcherHandlerRegister(KafkaFetcherConfig config) throws ModuleStart
         } else {
             isSharding = false;
         }
+        if (config.getKafkaHandlerThreadPoolSize() > 0) {
+            threadPoolSize = config.getKafkaHandlerThreadPoolSize();
+        }
+        if (config.getKafkaHandlerThreadPoolQueueSize() > 0) {
+            threadPoolQueueSize = config.getKafkaHandlerThreadPoolQueueSize();
+        }
+
+        enableKafkaMessageAutoCommit = (boolean) properties.getOrDefault(
+            ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, true);
         consumer = new KafkaConsumer<>(properties, new StringDeserializer(), new BytesDeserializer());
+        executor = new ThreadPoolExecutor(threadPoolSize, threadPoolSize,
+                                          60, TimeUnit.SECONDS,
+                                          new ArrayBlockingQueue(threadPoolQueueSize),
+                                          new CustomThreadFactory("KafkaConsumer"),
```

Comment:
If you need to reuse this, `CustomThreadFactory` should move from `org.apache.skywalking.oap.server.library.server.grpc` to `org.apache.skywalking.oap.server.library.server.pool`. Otherwise, the codes look like, Kafka is using gRPC pool.

Reply:
@wu-sheng Thank you for your suggestion.